src/server/ble_characteristic.rs

```diff
@@ -21,11 +17,7 @@ use crate::{
 };
 
 cfg_if::cfg_if! {
-  if #[cfg(all(
-    esp_idf_version_major = "5",
-    esp_idf_version_minor = "2",
-    not(any(esp_idf_version_patch = "0", esp_idf_version_patch = "1", esp_idf_version_patch="2"))
-  ))] {
+  if #[cfg(cpfd)] {
```

Comment:
This part is not related to cpfd.
The cpfd was changed in ESP-IDF version 5.2.1, but this part was changed in ESP-IDF version 5.2.3.

Reply:
Oh. Sorry. I'll push a new patch asap.

Reply:
Done.